KIPs/kip-249.md

```diff
@@ -279,9 +279,9 @@ The auctioneer is the independent node that relays the auction process. It valid
 Once the searchers deposit the KAIA into AuctionDepositVault, the auctioneer node will catch the event and recognize the searcher address.  
 The auctioneer node will implement the following json-rpc APIs for searchers.
 
-- `auction_submitBid`
-  - Input: `rlp([target_tx, block_num, sender, nonce, to_addr, call_gas_limit, call_data, bid_amount, from_signature])`
-  - Output: `(bid_hash, error)`
+- `http://<auctioneer-node-url>/api/v1/auction/send`
+  - Input: `{target_tx, block_num, sender, nonce, to_addr, call_gas_limit, call_data, bid_amount, from_signature}`
+  - Output: `{bid_hash, error}`
```

Comment:
There's another api `api/v1/subscriber/pendingtxs` which searchers can subscribe to the pending tx pool. Will KIP officially export only this API?

Reply:
Sorry for the late response.

I'm also not sure if we need to include other APIs in "KIP". Since they're too specific to auctioneer implementation, I think this is enough. How do you think?

Reply:
I Agree.
Raw bytes of target tx seem absent. (https://github.com/kaiachain/auctioneer/blob/main/api/rest_interface.go#L55)
(https://github.com/kaiachain/kaia/blob/feat/gasless-auction/kaiax/auction/impl/api.go#L122)

Reply:
Thanks. Added b3d2aec